p2p/host/resource-manager/obs/grafana-dashboards/resource-manager.json

```diff
@@ -626,7 +551,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "exemplar": true,
-          "expr": "histogram_quantile(0.50, (rcmgr_peer_streams_bucket - rcmgr_peer_streams_negative_bucket)) - 0.1",
+          "expr": "histogram_quantile(0.50, (rcmgr_peer_streams_bucket - rcmgr_previous_peer_streams_bucket)) - 0.1",
```

Comment:
Should we rename all the metrics to use the libp2p prefix?

Reply:
I'm torn. It's a good suggestion and makes sense. However, it would break any existing dashboards/tools that people have setup.

I decided to rename this one because I think the name was confusing or at least not very helpful.

Reply:
I think we should do it. Both swarm and eventbus metrics are using the libp2p prefix, would be good to be consistent. Let's do it in a follow-up PR though.